src/packaging/packages.targets

```diff
@@ -1,37 +1,18 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-    <UsingTask TaskName="GetBuildUid" TaskFactory="CodeTaskFactory" AssemblyName="Microsoft.Build.Tasks.v12.0">
-        <ParameterGroup>
-            <BuildUid ParameterType="System.String" Output="true" />
-        </ParameterGroup>
-        <Task>
-            <Code>
-                BuildUid = System.Guid.NewGuid().ToString().Replace("-", "").Substring(0, 8);
```

Comment:
Would it make sense to not use the global user cache instead to fix this problem? 

I do not like that each build will installs a new thing into the global cache. I have to regularly clean it up because of I end up with many hundreds of ILCompiler packages in it.
Would it make sense to not use the global user cache instead to fix this problem? 

I do not like that each build will installs a new thing into the global cache. I have to regularly clean it up because of I end up with many hundreds of ILCompiler packages in it.